df.units.xml

```diff
@@ -686,6 +686,7 @@
             <pointer type-name='unit' name='hunt_target'/>
             <pointer type-name='building' name='destroy_target'/>
 
+            <int32_t comment='v0.43.05' />
```

Comment:
Because that appears to be where Toady added a new member. anon_1 was -1, and it was pushed down. Granted, its semi-moot since we don't know what any of these are.